client/landing/subscriptions/tab-views/site-types.tsx

```diff
@@ -0,0 +1,9 @@
+// typo de dato con estos valores: id, name, icon, url, date, emailFrequency
```

Comment:
Could you delete this comment? Not necessary for this type

Reply:
Sure!